#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: fix ioperm build error on arm

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/en_US,v
retrieving revision 1.11
retrieving revision 1.12
diff -u -r1.11 -r1.12
--- libc/localedata/locales/en_US	2005/09/30 00:00:42	1.11
+++ libc/localedata/locales/en_US	2006/04/10 18:13:32	1.12
@@ -81,7 +81,7 @@
 	"<U0046><U0072><U0069><U0064><U0061><U0079>";/
 	"<U0053><U0061><U0074><U0075><U0072><U0064><U0061><U0079>"
 
-week    7;19971201;4
+week    7;19971130;7
 abmon	"<U004A><U0061><U006E>";"<U0046><U0065><U0062>";/
 	"<U004D><U0061><U0072>";"<U0041><U0070><U0072>";/
 	"<U004D><U0061><U0079>";"<U004A><U0075><U006E>";/
